app_legacy/Helpers/database/player-game.php

```diff
@@ -412,22 +412,22 @@ function getUsersCompletedGamesAndMax(string $user): array
     return legacyDbFetchAll($query)->toArray();
 }
 
-function getTotalUniquePlayers(int $gameID, ?string $requestedBy = null, bool $hardcoreOnly = false, ?int $achievementType = null): int
+function getTotalUniquePlayers(int $gameID, ?int $parentGameID = null, ?string $requestedBy = null, bool $hardcoreOnly = false, ?int $achievementType = null): int
 {
     $bindings = [
         'gameId' => $gameID,
     ];
 
-    $hardcoreStatement = '';
```

Comment:
I'm not sure these all need to be renamed to `base` since the mode/flags are not exclusive to just the base set when checking a subset

Reply:
You're right, it is certainly redundant now. Adjusted in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/e5b5a5c1e1bffe9a4fe9f3ff80f811756dc66590.